waiter/src/waiter/discovery.clj

```diff
@@ -17,13 +17,13 @@
            (org.apache.curator.framework CuratorFramework)))
 
 (defn- ->service-instance
-  [id svc-name {:keys [port]}]
+  [id svc-name {:keys [host port]}]
   (.. ServiceInstance
       builder
       (id id)
       (name svc-name)
       (uriSpec (UriSpec. "{scheme}://{address}:{port}/{endpoint}"))
-      ;(address host)
+      (address host)
```

Comment:
Does this work in the multi-router case? Also, does it work with the default of `0.0.0.0`?